src/DocRaptor.php

```diff
@@ -127,12 +148,22 @@ public function setStrict($strict = 'none')
     }
 
     /**
+     * @param boolean $javascript
+     * @return $this
+     */
+    public function setJavascript($javascript)
+    {
+        $this->javascript = $javascript;
+        return $this;
+    }
+
+    /**
      * @param bool $help
      * @return $this
      */
     public function setHelp($help = false)
     {
-        $this->help = (bool)$help;
+        $this->help = (bool) $help;
```

Comment:
My IDE generated the new setters ;) But actually, it's even not a good idea to cast to boolean here, the probability that this will end up true, is pretty high. I'll make a stricter check in the next PR.